docs/installation/installation-using-docker-linux.md

```diff
@@ -19,7 +19,7 @@ Take a look at the article about [Monorepo](../introduction/monorepo.md) for mor
 ## Steps
 ### 1. Create new project from Shopsys Framework sources
 ```
-composer create-project shopsys/project-base --stability=beta --no-install --keep-vcs
+composer create-project shopsys/project-base --no-install --keep-vcs
```

Comment:
please find "--stability=beta" in the all the md files in the project - there are still places where it was not removed (eg.there is a note with an explanation of the option below the command itself in each  of the guides, etc.)